cloud/src/autogluon/cloud/predictor/cloud_predictor.py

```diff
@@ -86,13 +95,19 @@ def __init__(
         """
         self.verbosity = verbosity
         set_logger_verbosity(self.verbosity, logger=logger)
-        self.role_arn = role_arn
-        if not self.role_arn:
-            self.role_arn = setup_sagemaker_role_and_policy(
-                role_name='ag_cloud_predictor_role',
-                trust_relationship=SAGEMAKER_TRUST_REPLATIONSHIP,
-                policies=SAGEMAKER_POLICIES
+        try:
+            self.role_arn = sagemaker.get_execution_role()
+        except ClientError as e:
+            logger.warning(
```

Comment:
Do we assume the user will run this with some form of Admin role. If not, can we provide the minimal set of policies required to the get the role setup?

Reply:
We are providing the util function for that